pkg/apis/componentconfig/v1alpha1/types.go

```diff
@@ -358,6 +358,9 @@ type KubeletConfiguration struct {
 	// rktPath is the  path of rkt binary. Leave empty to use the first rkt in
 	// $PATH.
 	RktPath string `json:"rktPath"`
+	// mounterPath is the path of mounter binary. If not set, kubelet will user default mount,
```

Comment:
nit: "path *to* mounter binary. kubelet will attempt to use `mount` binary that is available via `$PATH`.